internal/store/pod.go

```diff
@@ -196,6 +196,26 @@ var (
 				}
 			}),
 		},
+		{
+			Name: "kube_pod_restart_policy_always",
+			Type: metric.Gauge,
+			Help: "Set to 1 if the pod's restart policy is Always, 0 otherwise.",
+			GenerateFunc: wrapPodFunc(func(p *v1.Pod) *metric.Family {
+				var value float64
+				if p.Spec.RestartPolicy == v1.RestartPolicyAlways {
```

Comment:
I'm ok with only having a metric on this condition for now, but in order to allow potential expansion in the future, I'd prefer if we renamed this metric to `kube_pod_restart_policy` and make `restart_policy="Always"` a label, then still do the same with the value as this already does.

Reply:
Makes sense. So iiuc, is what you’re suggesting is the option 1 approach as laid out by @smarterclayton in #263.

Reply:
Yeah, that's why I went with this path - you sounded like you preferred option 2 :)

Reply:
Sorry about the confusion @smarterclayton :(.

Reply:
Yeah sorry, that was a misunderstanding.

Reply:
Np, updated.